scripts/start_ursim.sh

```diff
@@ -31,8 +31,17 @@
 PERSISTENT_BASE="${HOME}/.ursim"
 URCAP_VERSION="1.0.5"
 IP_ADDRESS="192.168.56.101"
-PORT_FORWARDING="-p 30001-30004:30001-30004 -p 29999:29999"
+PORT_FORWARDING_WITH_DASHBOARD="-p 30001-30004:30001-30004 -p 29999:29999"
+PORT_FORWARDING_WITHOUT_DASHBOARD="-p 30001-30004:30001-30004"
 CONTAINER_NAME="ursim"
+TEST_RUN=false
+
+# TODO: Add support for more URSim PolyScopeX versions once released
+# The PolyScopeX URSim containers follow the SDK versioning scheme. This maps those to marketing
+# versions
+#declare -Ag POLYSCOPE_X_MAP=( ["10.7.0"]="0.12.159"
+                             #["10.8.0"]="not.there.yet")
+declare -Ag POLYSCOPE_X_MAP=( ["10.7.0"]="0.12.159" )
```

Comment:
```suggestion
declare -Ag POLYSCOPE_X_MAP=( 
	["10.7.0"]="0.12.159" 
	["10.8.0"]="0.13.106" 
)
```
10.8.0 is available here https://hub.docker.com/r/universalrobots/ursim_polyscopex/tags

Reply:
I think you are right. So maybe it is just a 10.8.0.0😉

Reply:
Would you be OK with leaving this out of this PR for now?

Reply:
Yes, that is fine. Maybe we can consider how we are tagging the PolyScope X ursim